Automattic/wp-calypso
# Jetpack connect: Remove redux stateful redirections

This PR removes redux-stateful redirections `goBackToWpAdmin` and `goToXmlrpcErrorFallbackUrl` from Jetpack Connect.

These redux-stateful redirections only served to set a flag indicating redirection is in flight. It was up to consumers to respect the flag.

The redirection logic has been moved to the consuming components.

✅  ~Includes #21103 which isolates an alphabteization commit. For the "relevant bits", see 09553f8...9b457df~

## Follow-up

Move retryAuth out of redux, state via query

## Testing
1. Verify logged-in auth flows continue to work as expected.
1. Verify xml-rpc fallback works as expected:
   1.  Set console debug for Calypso:
      ```js
      localStorage.debug = 'calypso:jetpack-connect:authorize-form'
      ```
   1. Get to the authorize step with the full auth query in the url
   1. Break the site's `xmlrpc.php` (`mv xmlrpc.php xmlrpc.php_`)
   1. Click authorize.
   1. Should see xmlrpc debug output on the console (`xmlrpc fallback to …`)
   1. Should still work
